promises.cf

```diff
@@ -57,6 +57,10 @@ body common control
 
       version => "CFEngine Promises.cf 3.7.0";
 
+      cfengine_3_7.(debian||redhat)::
+          package_inventory => { $(package_module_knowledge.platform_default) };
+          package_module => $(package_module_knowledge.platform_default);
```

Comment:
Hmm, here's a problem: This will break 3.5 and 3.6. I thought we had handled this in the spec, but this apparently got left over. I don't immediately see how this can be solved, since splitting ``promises.cf`` is infeasible, and quite undesirable, regardless. Suggestions?